mscore/pianoroll/pianolevelschooser.ui

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>PianoLevelsChooser</class>
+ <widget class="QWidget" name="PianoLevelsChooser">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>369</width>
+    <height>288</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
```

Comment:
this string is better delete or make untranslatable, it isn't shown anywhere, so just confuses translators